device/mellanox/x86_64-nvidia_sn4280-r0/ACS-SN4280/sai.profile

```diff
@@ -0,0 +1,2 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4280.xml
+SAI_DEFAULT_SWITCHING_MODE_STORE_FORWARD=1
```

Comment:
Please define this as well as SAI_NOT_DROP_SIP_DIP_LINK_LOCAL=1 only in Mellanox-SN4280-O28 SKU